test/CMakeLists.txt

```diff
@@ -321,6 +323,16 @@ foreach(FILENAME ${test_input} ${test_output})
            ${CMAKE_CURRENT_BINARY_DIR}/${FILENAME} )
 endforeach(FILENAME)
 
+# create the share directory and link files to test subdir
```

Comment:
@srherbener   Please see if you agree this is an acceptable way to implement a link to the share directory and that all files in that directory are linked.  We could limit the fileglob to use file extensions like ```*.json``` if you think that is a better idea.